src/medCore/gui/toolboxes/medSelectorToolBox.cpp

```diff
@@ -35,17 +35,35 @@ medSelectorToolBox::medSelectorToolBox(QWidget *parent, QString tlbxId) :
     d->currentToolBox = NULL;
 
     d->chooseComboBox = new medComboBox;
-    d->chooseComboBox->addItem("Choose a toolbox");
-    d->chooseComboBox->setToolTip(tr("Choose a toolbox"));
+    d->chooseComboBox->addItem("* Choose a toolbox *");
+    d->chooseComboBox->setToolTip(tr("Choose a toolbox in the list"));
+    d->chooseComboBox->setItemData(0, "Choose a toolbox", Qt::ToolTipRole);
```

Comment:
this tooltip was missing, so if you put your cursor on the line (inside the combobox), there was either no tooltip, or the last one from the previous toolbox.